packages/browserify/examples/00-simple-cli/lavamoat/browserify/policy-override.json

```diff
@@ -0,0 +1,4 @@
+{
+  "resources": {
+  }
+}
```

Comment:
updated here like

https://github.com/LavaMoat/LavaMoat/blob/ba3b50d8732a23b7e50091f4dd8f62288916279c/packages/browserify/examples/01-simple-js/lavamoat/browserify/policy-override.json#L1-L4

Reply:
we could omit our generated policy files and rely on running `yarn lavamoat` in instructions

Reply:
added to both examples in https://github.com/LavaMoat/LavaMoat/pull/476/commits/4bd9640e58b761fce5435a2374e76671011475b6
but i noticed `yarn lavamoat` always removes missing EOF newlines in _policy.json_

Reply:
if we intend to be consistent can revert this commit
or follow-up with lavamoat cli to add missing newlines in both policy configs
(atm policy-override.json isn't formatted if it exists)